src/Lib/Managers/SnapManager.vala

```diff
@@ -0,0 +1,211 @@
+/*
+* Copyright (c) 2019 Alecaddd (https://alecaddd.com)
```

Comment:
This file was created in 2021, let's put that date here.
We use this approach to keep a visual track of when a file was initially created and consequentially updated.